.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
This file should be removed. If you need to keep it locally, you can run `git rm --cached -- .project`.